products/appengine/api.yaml

```diff
@@ -440,3 +440,39 @@ objects:
             name: 'shell'
             description: |
               The format should be a shell command that can be fed to bash -c.
+  - !ruby/object:Api::Resource
+    name: 'UrlDispatchRule'
+    description: |
+      Rules to match an HTTP request and dispatch that request to a service.
+    base_url: ''
+    create_url: 'apps/{{project}}?updateMask=dispatch_rules'
```

Comment:
What changes if we set `base_url` to `apps/{{project}}?updateMask=dispatch_rules` and don't set `create_url`/`delete_url`/`update_url`? I suspect it continues to generate the same thing. These fields *should* have default values.

Reply:
If we are setting the base_url and not setting others then test is failing. Tried with create_verb as well, getting either 404 or 400 code.

1 error(s) occurred:

* google_app_engine_application_url_dispatch_rules.rule: 1 error(s) occurred:

* google_app_engine_application_url_dispatch_rules.rule: Error creating ApplicationUrlDispatchRules: googleapi: got HTTP response code 404 with body: <h1>Not Found</h1>
<h2>Error 404</h2>


Error: Error applying plan:

1 error(s) occurred:

* google_app_engine_application_url_dispatch_rules.rule: 1 error(s) occurred:

* google_app_engine_application_url_dispatch_rules.rule: Error creating ApplicationUrlDispatchRules: googleapi: Error 400: This operation is only supported on the following field(s): [auth_domain, database_type, default_cookie_expiration, dispatch_rules, feature_settings, feature_settings.split_health_checks, feature_settings.use_container_optimized_os, serving_status]

Reply:
With base_url update its giving either 404 or 400 error, so keeping base_url blank and create_verb.